httpfuzzerprocessor/addCacheBusting.js

```diff
@@ -0,0 +1,31 @@
+function processMessage(utils, message) {
+	var cbValue = Math.floor(Math.random() * 10000)
+        setCacheBusting(message,cbValue);
+	message.getRequestHeader().setHeader("X-Cache-Busting", cbValue);
+}
+
+function setCacheBusting(message,cbValue) {
+    var URI = Java.type("org.apache.commons.httpclient.URI");
+    var params = message.getUrlParams();
+    var iterator = params.iterator();
+
+    if(iterator.hasNext()) {
```

Comment:
It would probably be simpler to just get the set of URL params and add the new one instead of worrying about setting the separator.

I’ll find an example.

Reply:
In https://github.com/zaproxy/community-scripts/blob/227558f4692d60eeda836cb1e1ca32e00bc841b0/httpsender/full-session-n-csrf-nashorn.js look for `setGetParams`

Reply:
Hi @kingthorin
First, Thank you for cool review. I Improve script code (using setGetParams)
If there's a better way, please let me know! Thanks to you, I learned one more good thing.

Reply:
Looks good to me, thanks for tackling that!